### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6799,6 +6803,8 @@ custumised->customised
 custumized->customized
 cutom->custom
 cuurently->currently
+cuve->curve
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: possibly also "cube", "cave"

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -6305,6 +6308,7 @@ coponent->component
 copoying->copying
 coppermines->coppermine
 coppied->copied
+copping->coping, copying,
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "cropping"

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -23423,6 +23438,7 @@ theer->there
 theery->theory
 theese->these
 thefore->therefore
+thei->their
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: possibly also "they"

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -24759,6 +24775,7 @@ unsychronised->unsynchronised
 untargetted->untargeted
 unter->under
 untill->until
+untils->until
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: possibly also "utils" (short for utilities)

**Comment by luzpaz** [comment]
@arm-in applied your feedback to 1ac93c8

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -6307,6 +6310,7 @@ coponent->component
 copoying->copying
 coppermines->coppermine
 coppied->copied
+copping->coping, copying, cropping,
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: `copping` is a form of the verb `cop`, e.g., "copping an attitude" (not entirely obscure in English)

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:6313]: good point. Though, this is why I for a 2ndary fallback dictionary accessible with a flag that can catch some of these more problematic typos (that are legit words)

Comment (larsoner) [reply codespell_lib/data/dictionary.txt:6313]: For now flag as `disabled ...` then? Then once we have the secondary dictionary support we can move it

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: @larsoner 5959efe  
rebased as well.

**Pull request merged**